packages/datx/src/types.ts

```diff
@@ -0,0 +1,3 @@
+export type PropertySelectorFn<T> = (item: T) => any;
```

Comment:
Hmm, this is a new thing... usually I handled types in the same way as interfaces (including the `I` prefix)

Reply:
Should I move this to `interfaces` folder and use `I` prefix?

Reply:
Yes, that would be great

Reply:
Fixed!